applogic/src/api/chat_details_cubit.rs

```diff
@@ -157,6 +157,25 @@ impl ChatDetailsCubitBase {
         }
         let editing_id = draft.and_then(|d| d.editing_id);
 
+        let salt: [u8; 16] = RustCrypto::default().random_array()?;
+        let content = if message_text == "delete" {
```

Comment:
This would delete a message when I just send `delete` as an edit, right? I think we need a boolean flag as a parameter.